### ISSUE TITLE
[BUG] Cannot join game through website via Bloxstrap

### ISSUE THREAD

**Issue by PeccNeck** [bloxstraplabs/bloxstrap#3186]
### Acknowledgement of preliminary instructions

- [X] I have read the preliminary instructions, and I am certain that my problem has not already been addressed.
- [X] I have thoroughly looked through the available Wiki articles and could not find a solution to my problem.
- [X] I am using the latest version of Bloxstrap.
- [ ] I did not answer truthfully to all the above checkboxes.

### What problem did you encounter?

Upon auto-updating to the latest version of Bloxstrap (2.8.0), I can no longer directly join games through the website. Attempting to do so simply opens Bloxstrap's menu; from there, I can still select Launch Roblox, but this just brings me into the Roblox app's home (which I don't want).

Steps to reproduce:
- In your browser, set `roblox` and `roblox-player` links to execute using Bloxstrap.
- On any game page, click either the green play button OR the Join button on any server.
- The Bloxstrap menu will open. You will not be sent into a game server.
- Click Launch Roblox.
- Bloxstrap will apply the modifications as normal.
- The Roblox app will open to the home page. You still will not be sent into a game server.

Not unplayable, but definitely inconvenient for someone like me who absolutely detests using the app for no reason in particular.

### Bloxstrap Log

N/A

**Comment by bluepilledgreat** [comment]
Can you send your log file?

**Comment by PeccNeck** [comment]
Can do, I've got two of them.
[Bloxstrap_20241016T181110Z.log](https://github.com/user-attachments/files/17400950/Bloxstrap_20241016T181110Z.log) - this one was generated first when the menu opened.
[Bloxstrap_20241016T181117Z.log](https://github.com/user-attachments/files/17400951/Bloxstrap_20241016T181117Z.log) - generated when I selected Launch Roblox.

Bloxstrap was launched from a website join link for these; I did not manually join a game after selecting Launch Roblox, I simply closed the window.

**Comment by pizzaboxer** [comment]
I'm still not entirely clear on this, can you send a video of you attempting to launch from the website?

**Comment by PeccNeck** [comment]
https://github.com/user-attachments/assets/7832cad6-ae3d-4e7e-a203-5096ff393be8

It'll launch Bloxstrap, it'll launch Roblox, but it won't actually join the game.
![image](https://github.com/user-attachments/assets/7df60b3e-3e8d-4dcf-89ad-045e32544896)

**Comment by pizzaboxer** [comment]
Can you open the Windows Registry Editor, go to `HKEY_CURRENT_USER\SOFTWARE\Classes\roblox-player\shell\open\command`, and show a screenshot of what the values shown there are?

**Comment by PeccNeck** [comment]
![image](https://github.com/user-attachments/assets/d5edf5a6-b81a-4b9b-8ccf-c45dc905368d)

**Comment by PeccNeck** [comment]
![image](https://github.com/user-attachments/assets/5ffba17c-7eb2-48c2-bfa6-b2e36fae1770)

**Comment by pizzaboxer** [comment]
Attempt a Roblox launch from the website again. Make sure you don't close it.

Then, open a PowerShell window and run this command:

```powershell
Get-WmiObject Win32_Process -Filter "name = 'Bloxstrap.exe'" | Select-Object CommandLine
```

Send a screenshot to me of what you see privately (email me at pizzaboxer@proton.me), let me know here when you have.

**Comment by PeccNeck** [comment]
> Send a screenshot to me of what you see privately (email me at [pizzaboxer@proton.me](mailto:pizzaboxer@proton.me)), let me know here when you have.

Went ahead and sent it; email made the image a bit small, might need to open it in a new tab.

It definitely looks like Bloxstrap is receiving the launch options, but... nothing happens.

**Comment by pizzaboxer** [comment]
This strangely seems like a Firefox issue, since it's not actually obeying the rules specified in the Registry. I'll come up with a workaround for this.

**Comment by PeccNeck** [comment]
Really? That's especially strange; I've been using Firefox for a while and I've never had an issue like this with anything else, not even older versions of Bloxstrap.

Oh well.

**Comment by bluepilledgreat** [comment]
What arguments is Bloxstrap launched with?

**Comment by pizzaboxer** [comment]
> What arguments is Bloxstrap launched with?

It's acting as if it's "Bloxstrap.exe %1" - I'll add an exception in the launch settings parser for it

**Comment by pizzaboxer** [comment]
To get around this for the time being, use this prerelease version of 2.8.1: https://github.com/bloxstraplabs/bloxstrap/actions/runs/11526434565/artifacts/2106464097

**Comment by ProjectHSI** [comment]
not having this problem - even without the prerelease version

bit weird why some firefox users are having this problem and others aren't (i.e. me)...

**Comment by PeccNeck** [comment]
> To get around this for the time being, use this prerelease version of 2.8.1: https://github.com/bloxstraplabs/bloxstrap/actions/runs/11526434565/artifacts/2106464097

Prerelease seems to be working for the time being; will update if any issues crop up. Thank you!

**Comment by pizzaboxer** [comment]
Fixed in 2.8.1 - reopen if not
